app/assets/javascripts/courseEdit/components/SlideForm.js

```diff
@@ -85,19 +72,18 @@ class SlideForm extends React.Component {
             />
           </div>
 
-          {this.renderImage()}
-
           <div className='add-component-form-item'>
             <ImageUploadInput
-              label="Image"
-              onChange={this.updateImageData}/>
+              label='Image'
+              image={this.state.component.imageData || this.state.imageUrl}
+              setImage={this.updateImageData}/>
           </div>
 
           {this.renderAudio()}
 
           <div className='add-component-form-item'>
             <AudioUploadInput
-              label="Audio"
+              label='Audio'
```

Comment:
Nit: using airbnb's style guide use double quotes for all react props and single quotes for everything else